apps/ui/src/views/Space/Settings.vue

```diff
@@ -495,18 +495,10 @@ watchEffect(() => setTitle(`Edit settings - ${props.space.name}`));
         title="Controller"
         description="The controller is the account able to change the space settings and cancel pending proposals."
       >
-        <UiMessage
```

Comment:
Also need to edit the message in EnsConfiguratorOffchain.vue

Reply:
The message is true though 🤔 During space creation owner will be the controller

> Your space controller will be set to the ENS name owner. Any changes
          to the ENS name ownership will also change the controller.

Reply:
Any suggestions on text change?

Reply:
But this field is read-only 😄  so second part Is kind of true, controller will be changed if ownership is changed. 

Do we really need to show controller here? 🤔 

cc @bonustrack

Reply:
Should it be something like this?

> The controller is configured on ENS name record. To change the controller, you need to be owner of the ENS name.

This clarifies that controller isn't really owner of ENS name (as we can have `snapshot` record on domain that doesn't have to match the owner of the name).
This also removes inaccurate line that you need to change owner of ENS name to change controller (same as before, we change it by setting `snapshot` record).

I think first sentence can be improved, I'm not sure it captures the meaning that it can be either the `snapshot` record on the name or defaults to the ENS name's owner.

Reply:
Yep, missing context that ENS owner is controller by default

How about
> By default, the ENS domain’s controller is its owner. However, the owner can assign a different controller by updating the snapshot text record.

Reply:
Or just defer the explanation to later, and just show in the space creation:

> By default, the ENS domain’s controller is its owner. You can change it later in your space setting.

This will simplify the flow, we just explain why it is set to this value here, any other explanation about how it works will come later.

Then in the space setting, we keep the UI message, with something like that, only for the controller, if he's not also the owner:

> Controller can only be edited by the ENS owner

This message will remove confusion about why some controller can not edit the controller for offchain spaces, as opposed to onchain spaces.

Reply:
Done updated